_pages/castrosalazr.yaml

```diff
@@ -3,4 +3,3 @@ githubHandle: castrosalazr
 pageUrl: castrosalazr.github.io
 timestamp: 2017-06-29
 ---
-
```

Comment:
Hi @jenlscribner! It looks like this pull request is failing because of a few errors in this file. I'm going to go ahead and correct them since your file looks 💯.